superset/databases/schemas.py

```diff
@@ -555,6 +563,12 @@ class SchemasResponseSchema(Schema):
     result = fields.List(fields.String(description="A database schema name"))
 
 
+class DatabaseTablesResponse(Schema):
+    extra = fields.Dict(required=False)
+    type = fields.String()
+    value = fields.String()
```

Comment:
would be nice to add a description on these fields to improve out documentation for the API